airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/surrogate_key.sql

```diff
@@ -0,0 +1,12 @@
+{# column_quote ----------------------------------     #}
+{% macro QUOTE(column_name) -%}
```

Comment:
Should this override surrogate_key macro? 
the file is currently the same as airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/quote.sql ?

Reply:
I removed this and use default ORA_HASH to hash values.